server/conf/docker/docker-compose-dev.all.debug.yml

```diff
@@ -31,6 +31,9 @@ services:
   memcached:
     image: memcached:latest
     container_name: core_portal_memcached
+    ports:
+      - 127.0.0.1:11211:11211
+    command: ["memcached"]
```

Comment:
Since this is local, do we need to apply these changes to Camino as well?
https://github.com/TACC/Camino/blob/main/conf/compose/docker-compose.memcached.yml